projectInfo.js

```diff
@@ -6,20 +6,20 @@ var projectInfo = [
     dateCreated: '07-10-2016',
     image: 'imgs/guess.jpg', //create imgs folder... put image filepath here.
     description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
-    // fileSource: '', //project filepath
+    //fileSource: 'https://kaymay1000.github.io/guessing_game/',
   },
   {
     title: 'Campfire Coffee',
     dateCreated: '08-10-2016',
     image: 'imgs/coffee.jpg',
     description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
-    // fileSource: '',
+    //fileSource: 'https://kaymay1000.github.io/campfire_coffee/',
   },
   {
     title: 'Bus Mall',
     dateCreated: '08-20-2016',
     image: 'imgs/busmall.jpg',
     description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
-    // fileSource: '',
+    // fileSource: 'https://github.com/kaymay1000/bus_mall',
   },
 ];
```

Comment:
I spent some time debugging your code and it seems that there is a problem with your JSON. Not sure what it is, but you might want to try using JSON.stringify'ing it in the browser or using a utility online that will make valid JSON for you. That's the reason your AJAX call is silently failing (at least one reason).